datatypes/src/util/mod.rs

```diff
@@ -5,6 +5,7 @@ pub mod helpers;
 pub mod identifiers;
 pub mod ranges;
 mod result;
+pub mod retry;
```

Comment:
Shoudln't this rather be part of operators?

(Then, we could remove futures and tokio from datatypes again.)

Reply:
92f3e595cc959be5f89d1faaf008eb4179ef9852